examples/get_cert.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 #
-# Copyright 2019 Venafi, Inc.
+# Copyright 2022 Venafi, Inc.
```

Comment:
should it not 2021?

Reply:
There's only one month left for 2021. Im not coming back to change it when 2022 starts in one month. Im changing it now because Im already working on these files.